lms/djangoapps/discussion/static/discussion/js/spec/discussion_board_view_spec.js

```diff
@@ -0,0 +1,408 @@
+/* globals _, DiscussionUtil, setupAjax, Thread */
+define(
+    [
+        'jquery',
+        'backbone',
+        'common/js/spec_helpers/page_helpers',
+        'common/js/spec_helpers/discussion_spec_helper',
+        'edx-ui-toolkit/js/utils/constants',
+        'discussion/js/views/discussion_board_view',
+        'discussion/js/views/discussion_search_view'
+    ],
+    function($, Backbone, PageHelpers, DiscussionSpecHelper, constants, DiscussionBoardView, DiscussionSearchView) {
+        'use strict';
+
+        xdescribe('Discussion Board View', function() {
+            var initializeDiscussionBoardView = function() { // eslint-disable-line no-unused-vars
+                DiscussionBoardView({
+                    el: $('.discussion-board'),
+                    courseId: 'test_course_id',
+                    course_name: 'Test Course',
+                    user_info: DiscussionSpecHelper.getTestUserInfo(),
+                    roles: DiscussionSpecHelper.getTestRoleInfo(),
+                    sort_preference: null,
+                    threads: [],
+                    thread_pages: [],
+                    content_info: null,
+                    course_settings: {
+                        is_cohorted: false,
+                        allow_anonymous: false,
+                        allow_anonymous_to_peers: false,
+                        cohorts: [],
+                        category_map: {}
+                    }
+                });
+            };
+
+            beforeEach(function() {
+                PageHelpers.preventBackboneChangingUrl();
+
+                // Install the fixtures
+                setFixtures(
+                    '<div class="discussion-body">' +
+                    '    <div class="forum-nav"></div>' +
+                    '</div>'
+                );
+                DiscussionSpecHelper.setUnderscoreFixtures();
+            });
+
+            afterEach(function() {
+                Backbone.history.stop();
+            });
+
+            describe('browse menu', function() {
+                var expectBrowseMenuVisible;
+
+                expectBrowseMenuVisible = function(isVisible) {
+                    expect($('.forum-nav-browse-menu:visible').length).toEqual(isVisible ? 1 : 0);
+                    return expect($('.forum-nav-thread-list-wrapper:visible').length).toEqual(isVisible ? 0 : 1);
+                };
+
+                it('should be visible by default', function() {
+                    expectBrowseMenuVisible(true);
+                });
+
+                describe('when shown', function() {
+                    it('should show again when header button is clicked', function() {
+                        $('.forum-nav-browse').click();
+                        return expectBrowseMenuVisible(false);
```

Comment:
This looks like it may have been converted from Coffee at some point? All of these returns (68, 73, 78, 83, etc) are unnecessary